_pages/platform/usage-metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: article
+title:  "Usage Metrics"
+excerpt: "The Admin Panel's Usage Feature"
+categories: basics
+nav_index: 40
+tags: [basics, alg-dev-getting-started, app-dev-getting-started]
+show_related: true
+image:
+    teaser: /icons/algo.svg
+permalink: /platform/usage/
+redirect_from:
+  - /basics/usage/
+---
+
+
+### Platform Usage Metrics
```

Comment:
Let's update this to `## Platform Usage Metrics` so it ends up rendering as an `<h2>`